src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ set(NATIVEGLOBALIZATION_SOURCES
     pal_normalization.c
     pal_timeZoneInfo.c
     pal_icushim.c
+    entrypoints.c
```

Comment:
The reason of this file being in libraries-native was to have the infrastructure in the future to link more native shims into the runtime, why did we need to move it?

Reply:
https://github.com/dotnet/runtime/pull/37670#discussion_r438900799
> `ICU_SHIM_PATH` may be a good place for this one. I agree that we need a better to share C impls between CoreCLR and Mono.

Reply:
Yeah discussed offline, this makes sense as is.